src/manifest.js

```diff
@@ -261,22 +261,37 @@ export const addPropertiesToMaster = (master, uri) => {
   }
 
   forEachMediaGroup(master, (properties, mediaType, groupKey, labelKey) => {
-    if (!properties.playlists ||
-        !properties.playlists.length ||
-        properties.playlists[0].uri) {
-      return;
+    const groupId = `placeholder-uri-${mediaType}-${groupKey}-${labelKey}`;
+
+    if (!properties.playlists || !properties.playlists.length) {
+      properties.playlists = [Object.assign({}, properties)];
     }
```

Comment:
This may impact this check: https://github.com/videojs/http-streaming/blob/552b012035b6f2fd8da8440a41928668d85e25a8/src/media-groups.js#L471

Which looks for the existence of playlists.

Reply:
I think this might be okay since it is looking for dash sourceType specifically here and those will always have playlists. It does change our notions around a bit though.

Reply:
If DASH always has `playlists` in this case, I wonder why I included that case at all back when.

Reply:
maybe it didn't at the time this was added

Reply:
Moved the code to `updateMaster` where we update the media group playlist with changes from the requested playlist (usually just segments)